aws/table_aws_cloudtrail_lake_query.go

```diff
@@ -0,0 +1,199 @@
+package aws
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail/types"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudtrailLakeQuery(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_cloudtrail_lake_query",
```

Comment:
Should it be `aws_cloudtrail_query` instead? Looking at the API, there's no mention of `Lake` (https://docs.aws.amazon.com/awscloudtrail/latest/APIReference/API_ListQueries.html), though other AWS docs do associate them with CloudTrail Lake (https://docs.aws.amazon.com/awscloudtrail/latest/userguide/query-create-edit-query.html).

Reply:
Updated the table name.